### PR TITLE
Wishlist functionality react

### PR DESCRIPTION
**Pull request by mayuresh-tech**

### CODE REVIEW

File: src/components/listing/ListingPage.jsx [hunk 1]
```diff
@@ -1,170 +1,75 @@
...
 
-const ListingPage = ({ item }) => {
+const ListingPage = () => {
+  const { data } = useData();
+
+  let trendingArr = [];
+  let productsArr = [];
+
+  console.log(trendingArr);
+  console.log(productsArr);
```
Comment (BALAJI141031) [inline src/components/listing/ListingPage.jsx]: you can remove logs if its working fine

File: src/components/listing/ListingPage.jsx [hunk 2]
```diff
@@ -1,170 +1,75 @@
...
-            closeActive: false,
-            wishlisted: false,
-            imagePath: "/assets/product.jpg",
-          }}
-        />
+        {productsArr.map((item) => {
+          return (
+            <ProductCardVertical
+              key={item._id}
+              item={{
```
Comment (BALAJI141031) [inline src/components/listing/ListingPage.jsx:53]: what is the intention to create object to pass prop?i guess instead you can pass item directly and in destinated component you can destructure whate ever properties of that item you want

**Review by BALAJI141031 — APPROVED** [review]

File: src/components/listing/ListingPage.jsx [hunk 3]
```diff
@@ -1,170 +1,74 @@
...
+              trendingText: item.trendingText,
+              categoryName: item.categoryName,
+              soldBy: item.soldBy,
+              rating: item.rating,
+              salePrice: item.salePrice,
+              originalPrice: item.originalPrice,
+              discountPercent: item.discountPercent,
+              closeActive: item.closeActive,
+              wishlisted: item.wishlisted,
+              imagePath: item.imagePath,
```
Comment (deekshithmd) [inline src/components/listing/ListingPage.jsx:42]: Check whether it is good or destructuring item in component is good...

**Review by deekshithmd — APPROVED** [review]

File: src/components/listing/ListingPage.jsx (hunk 1 above)
Comment (mayuresh-tech) [reply src/components/listing/ListingPage.jsx]: Removed in next feature

File: src/components/listing/ListingPage.jsx (hunk 2 above)
Comment (mayuresh-tech) [reply src/components/listing/ListingPage.jsx:53]: Will look into this and update in fixes feature

File: src/components/listing/ListingPage.jsx (hunk 3 above)
Comment (mayuresh-tech) [reply src/components/listing/ListingPage.jsx:42]: Will look into this and update in fixes feature

**Pull request merged**